src/Tools/Test Framework/Test Runner/src/DataDrivenTest/DataOutputs/TestOutput.Table.al

```diff
@@ -0,0 +1,82 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.TestRunner;
+
+table 130453 "Test Output"
+{
+    DataClassification = CustomerContent;
+    TableType = Temporary;
+
+    fields
+    {
+        field(1; "Test Suite"; Code[10])
```

Comment:
nit: I'd expect the name "Test Suite Name" (or Code) here.

Reply:
It is aligned with the Test Method Line. I will keep it for now even though it is bad.